fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordCursorContinuation.java

```diff
@@ -60,6 +62,16 @@
     @Nullable
     byte[] toBytes();
 
+    /**
+     * Serialize this continuation to a ByteString object.
+     * @return a (possibly EMPTY) ByteString containing a binary serialization of this continuation
+     */
+    @Nonnull
+    default ByteString toByteString() {
+        final byte[] bytes = toBytes();
+        return bytes == null ? ByteString.EMPTY : ByteString.copyFrom(bytes);
```

Comment:
I'm not sure I'm sold if this is the right behavior if `toBytes()` returns `null`, though I suspect that most of the time, an empty and a null byte string should have the same semantics. The alternative, I think, would be to have this method always return `null` if `toBytes()` is `null`, and then a different method, `toByteStringOrEmpty` could return `ByteString.EMPTY` if this one returns `null`. But maybe that's needlessly complicated.

In any case, we should document the behavior (that if `toBytes` is `null`, then this returns `EMPTY`).

Reply:
Added comments for this method. Please let me know if it looks good to you.